engine/src/main/java/org/hibernate/search/engine/service/impl/StandardServiceManager.java

```diff
@@ -221,7 +221,7 @@ synchronized void stopVirtual() {
 				if ( status != ServiceStatus.RUNNING ) {
 					stateExpectedFailure();
 				}
-				stopAndRemoveFromCache();
+				stopReal();
```

Comment:
Take it as a testimony that all this stuff was really not meant to do service start/stop frequently at runtime ;-)